content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,63 @@
+
+> **Note:** Contrast agents make a “best effort” attempt to avoid printing sensitive data in debug log statements; however, it’s possible that sensitive data could appear in the Contrast log, if the log level is set to DEBUG or lower. Whenever possible, you should avoid setting production systems to log at DEBUG or lower. If a system that deals with sensitive data is set to log at DEBUG or lower, you should take steps to ensure that those logs are not being sent to an external system to avoid leaking any sensitive data. 
+
+#### Example
+
+The following sample HTTP request sent by an agent as part of a vulnerability report shows two inputs that the agent identified as sensitive as well as the placeholders it used to mask the values of the input before sending the report to the Contrast UI, syslog server or security log. 
+ 
+```
+POST/employee/5 HTTP 1.1 
+
+Host: example.com 
+
+Content-Type: application/x-www-form-urlencoded 
+
+Content-Length: 30 
+
+Authorization: contrast-redacted-authentication 
+ 
+
+ssn=contrast-redacted-id&department=sales 
+
+``` 
+
+In this case, the header value is masked because "Authorization" matches a keyword in the "Authentication" data type, and the form parameter is redacted because "ssn" matches a keyword in the "ID" data type for your Contrast organization. (Keyword matches are case insensitive.) <!-- TODO: are the above placeholder data types and keywords actual real types/keywords for this feature?  -->
+
+## Manage Sensitive Data Types
+
+Go to the **user menu > Policy Management > Sensitive Data tab** to view and edit the sensitive data types for your organization. In the **Sensitive Data** grid, data types and associated keywords are listed alphabetically. To quickly find data types by name or keywords, use the search field above the grid. 
+
+To enable redaction of the entire HTTP request entity, check the box to **Mask entire body**. This will apply to all applications in your organization.
+
+<a href="assets/images/Sensitive-data-grid.png" rel="lightbox" title="View sensitive data types and keywords in the grid"><img class="thumbnail" src="assets/images/Sensitive-data-grid.png"/></a>
+
+Critical data types and keywords determined by Contrast apply to all applications in your organization by default, and can't be edited or disabled. For data types that Contrast has not determined to be critical, you may use the toggle in the grid to enable or disable them for the organization.
+
+### Edit data types
+
+Click on the name of the data type in the grid to add custom keywords. In the **Edit Sensitive Data Type** form, use the **Custom Keywords** field to add more keywords and specify the applications to which they apply. **Default Keywords** aren't editable, and apply to all applications. 
+
+> **Note:** Keywords must be alphanumeric and contain no spaces; agent identification is case insensitive. 
```

Comment:
This keyword restriction doesn't apply anymore - we do not restrict what someone can use for a keyword.  The case insensitive part is still true though.